packages/ui/src/working-groups/queries/workingGroups.graphql

```diff
@@ -101,8 +101,8 @@ query GetWorkingGroupOpeningQuestions($id: ID!) {
   }
 }
 
-query GetWorkingGroup($id: ID!) {
-  workingGroupByUniqueInput(where: { id: $id }) {
+query GetWorkingGroup($name_eq: String) {
+  workingGroupByUniqueInput(where: { name_eq: $name_eq }) {
     ... WorkingGroupFields
```

Comment:
I think that this one will be to change so the other `name_eq` in the code may left intact